package.json

```diff
@@ -74,7 +74,8 @@
     }
   },
   "sideEffects": [
-    "*.css"
+    "*.css",
+    "./src/i18n/Streami18n.ts"
```

Comment:
I think it should be `./dist/i18n/Streami18n.js` :)

Reply:
Fixed :)